File: Data Structures & Algorithms/duplicate-integer/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {

        let count = {}; 

        for(let item of nums){
          if(count[item]){
            return true; 
          }else{
            count[item] = 1
          }
        }return false



    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
        Set<Integer> seen = new HashSet<>();
        

        for(int num: nums){
            seen.add(num);
        }

        int longest = 0; 
        
        for(int num: nums){
            int count = 1;
            int currNum = num + 1; 
            while(seen.contains(currNum)){
                count++;
                currNum++;
            }
            longest = Math.max(longest, count);
        }

        return longest;
    }
}
